src/components/MessageList/hooks/useScrollLocationLogic.tsx

```diff
@@ -57,7 +58,14 @@ export const useScrollLocationLogic = <
 
   useEffect(() => {
     if (!listElement) return;
-    const observer = new ResizeObserver(scrollToBottom);
```

Comment:
@MartinCupela the resizeobserver was very trigger happy, throwing events even when the height was not changed (for example, when gallery was open).